### ISSUE TITLE
delta in _process is incorrect when force_fps is low

### ISSUE THREAD

**Issue by tinyleolin** [godotengine/godot#24624]
**Godot version:**

Godot Engine v3.0.6.stable.official.8314054 (installed from Steam)

**OS/device including version:**

Windows 10 version 1803 (OS internal version 17134.471)

**Issue description:**

When setting `force_fps` to 2, the `delta` in `_process(delta)` is far smaller (which is **0.133333**) than the actual time period between two frames (which is **0.5**).

**Steps to reproduce:**
Just set the `force_fps` to 2, and `print(delta)` in `_process`.

**Comment by tinyleolin** [comment]
#5332 Setting `force_fps` to 2 also makes the editor very laggy after reopening the project.

**Comment by timoschwarzer** [comment]
I'm not sure if `_process` is even guaranteed to only run once per frame. I think it's guaranteed to run _at least_ once per frame.

**Comment by tinyleolin** [comment]
@timoschwarzer In my test case, `_process` is called only once per frame, but `delta` does not equal to the period between two frame.
Just enable `print_fps` and you can see the `print(delta)` output and "Game FPS: 2" showing up in pair.

**Comment by timoschwarzer** [comment]
@tinyleolin Thanks for clarification. I can reproduce this issue in 3.1-alpha4. @akien-mga 	

```gdscript
func _process(delta):
	print(str(OS.get_ticks_usec() / 1000000.0) + " - " + str(delta))
```

Output:
![image](https://user-images.githubusercontent.com/8849554/50539603-63882980-0b83-11e9-8aca-daad99c697b3.png)

**Comment by Toshiwoz** [comment]
I am experiencing this too, i my case it's affecting tween duration, if I force FPS to any value below my computer's limit it affects it (I think, but it's not proven) proportionally to the difference between the actual machine FPS limit and the forced one.
In other words, if force FPS is a very high value, say 900 FPS, it should not affect, but if I set 60 it does change the tween time noticeably (for example instead of lasting 2 secs it lasts 0.2).

In my case this feature is needed because when I do screen recording, without an FPS limit, Godot does not leave enough CUP to OBS or S>G.

**Comment by TheDeafCreeper** [comment]
I would just like to add that this seems to occurs even with just a low fps even if not set (in my case less then 1).

**Comment by aaronfranke** [comment]
0.13333 time elapsed per frame would mean 7.5 FPS. This issue may have something to do with the limitation of Godot where physics tickrate is capped at 8 times the game's framerate, since 60 / 8 = 7.5. What happens if you change the physics tickrate (Project settings -> Physics -> Common -> Physics FPS)?

**Comment by ludomosa** [comment]
This is still an issue.

Set the physics FPS to 240, then force FPS to any value below 30; watch as _process delta never dips below 0.0333 (stuck at 1/30th of a second, or 8x that of physics delta.)

If this is intentional, then a warning should be produced so the user knows that delta is incorrect, and this 8x relationship  should be documented. Otherwise things just seem to slow down with no clear reasoning as to why.

**Comment by Zireael07** [comment]
@ludomosa: I believe the 8x part has recently been documented.

**Comment by Calinou** [comment]
Closing in favor of https://github.com/godotengine/godot-proposals/issues/1893, as feature proposals are now tracked on the Godot proposals repository. (The current behavior is done by design to prevent the engine from freezing due to heavy physics calculations at low FPS.)
